src/init-ceph.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
@tchaikov The rationale here was that we are not supporting any non-Linux platforms, so why not be explicit about that?